web/thesauruses/c/17/io.json

```diff
@@ -0,0 +1,59 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "io"
+    },
+    "concepts": {
+        "write_line": {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
```

Comment:
You can also use `\"` as a way to escape quotation marks so they still show up. You shouldn't use the "fancy" quotes as if someone copied and pasted this code, it wouldn't work since it's not a valid part of the language.